Index: stdeb.cfg
===================================================================
--- stdeb.cfg	(revision 3412)
+++ stdeb.cfg	(working copy)
@@ -1,5 +1,5 @@
 [Sutekh]
 # The librsvg2-common dependency is to ensure that the gdk-pixbuf loader for
 # SVG images is installed
-Depends: librsvg2-common, python-gtk2, python-configobj, python-ply, python-singledispatch, python-keyring
+Depends: librsvg2-common, python-gtk2, python-sqlobject, python-configobj, python-ply, python-singledispatch, python-keyring
 Copyright-File: COPYRIGHT
Index: sutekh/SutekhInfo.py
===================================================================
--- sutekh/SutekhInfo.py	(revision 3411)
+++ sutekh/SutekhInfo.py	(working copy)
@@ -79,8 +79,6 @@
     ]
 
     INSTALL_REQUIRES = [
-        'SQLObject >= 0.9.0, < 3.4',  # fetching the dev version requires svn
-                           # (which is a bit crazy as an install requirement)
         'singledispatch',
         'ply',
         'configobj',
