Index: stdeb.cfg
===================================================================
--- stdeb.cfg	(revision 3412)
+++ stdeb.cfg	(working copy)
@@ -1,5 +1,5 @@
 [Sutekh]
 # The librsvg2-common dependency is to ensure that the gdk-pixbuf loader for
 # SVG images is installed
-Depends: librsvg2-common, python-gtk2, python-configobj, python-ply, python-keyring
+Depends: librsvg2-common, python-gtk2, python-sqlobject, python-configobj, python-ply, python-keyring
 Copyright-File: COPYRIGHT
Index: sutekh/SutekhInfo.py
===================================================================
--- sutekh/SutekhInfo.py	(revision 4184)
+++ sutekh/SutekhInfo.py	(working copy)
@@ -86,7 +86,6 @@
     ]
 
     INSTALL_REQUIRES = [
-        'SQLObject >= 3.0.0',
         'ply',
         'configobj',
